admin_guide/out_of_resource_handling.adoc

```diff
@@ -60,6 +60,31 @@ When the node fails a pod, it terminates all containers in the pod, and
 the link:https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase[`PodPhase`] is transitioned to *Failed*.
 ====
 
+The node supports the `nodefs` and `imagefs` file system partitions when detecting disk pressure, as follows:
+
+* The `nodefs`, or `rootfs`, file system that the node uses for local disk volumes, daemon logs, emptyDir,
+and so on (for example, the file system that provides `/`). Evictions do not apply for nodefs.
```

Comment:
evictions do apply to nodefs (though I might have told you otherwise in a moment of confusion)

https://kubernetes.io/docs/tasks/administer-cluster/out-of-resource/#eviction-policy